utils/awesome-client

```diff
@@ -7,11 +7,11 @@ set -o pipefail
 
 if [ -t 0 ]  # is a tty.
 then
-    # rlwrap provides readline functionality for "read", which is more enhanced
+    # rlwrap provides readline functionality for "read", command -v is more enhanced
```

Comment:
What @actionless is probably hinting at:
It looks like you just ran a "replace all", which also affected this comment.